src/components/Animals.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div class="mainContainer">
+    <v-app>
+      <v-main>
+        <ul class="header">
+          <li>
+            <span class="heading"><h2>Animals of the Kingdom</h2></span>
+          </li>
+          <li><Form /></li>
+        </ul>
+        <v-container>
+          <v-row no-gutters>
+            <v-col
+              v-for="(item, index) in animals"
+              :key="index"
+              cols="12"
+              sm="6"
+              md="4"
+              lg="4"
+              xs="12"
+            >
+              <v-card class="mx-auto thumbnail" width="290.5px">
+                <v-img
+                  class="white--text align-end"
+                  max-height="200px"
+                  v-bind:src="item.src"
+                >
+                  <v-card-actions class="close">
+                    <v-icon left @click="remove(index)">
+                      mdi-close
+                    </v-icon>
+                  </v-card-actions>
+                </v-img>
+                <v-card-title>{{ item.name }}</v-card-title>
+              </v-card>
+            </v-col>
+          </v-row>
+        </v-container>
+      </v-main>
+    </v-app>
+  </div>
+</template>
+
+<script>
+import Form from "./Form";
+import { mapGetters } from "vuex";
+export default {
+  name: "App",
+
+  components: {
+    Form,
+  },
+  computed: {
+    ...mapGetters({
+      animals: "allAnimals",
+    }),
+  },
+  methods: {
+    create() {
+      this.$store.state.showModal = true;
+    },
+    remove(index) {
+      this.$store.commit("remove", index);
+    },
+  },
+};
+</script>
+<style>
```

Comment:
why this much css being used for this simple layout also we don't need additional media query in this
when using component library we avoid media query as much as we can
instead break points provided by vuetify or respected component library